python/get_user_watchlist_feed.py

```diff
@@ -0,0 +1,29 @@
+#This file is partly auto-generated
+
+#!/usr/bin/python3
+
+"""
+    feedwatchlist_other.py
```

Comment:
Filename in the code is different than the actual.

Reply:
Fixed in https://github.com/srish/MediaWiki-Action-API-Code-Samples/pull/42/commits/6d0834180dc5c959cb016f363a1c0eb046357ba5